### PR TITLE
[ENT] Enable Redis for JWT and tenant settings cache by default

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: title

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by 0lmi** [comment]
LGTM

**Review by 0lmi — APPROVED** [review]

**Review by kjaskiewiczz — APPROVED** [review]

File: testutils/infra/container_manager/docker_compose_manager.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ class DockerComposeNamespace(DockerNamespace):
     ]
 
     NUM_SERVICES_OPENSOURCE = 13
-    NUM_SERVICES_ENTERPRISE = 16
+    NUM_SERVICES_ENTERPRISE = 17
```
Comment (lluiscampos) [inline testutils/infra/container_manager/docker_compose_manager.py:112]: We need a similar update in the mender-docs tests See `EXPECTED_COUNT_ENTERPRISE` in [here](https://raw.githubusercontent.com/mendersoftware/mender-docs/master/07.Server-installation/03.Installation-with-docker-compose/02.Upgrading-from-OS-to-Enterprise/docs.md).

**Pull request merged**